modules/wri_io_content/wri_io_content.install

```diff
@@ -2,14 +2,25 @@
 
 /**
  * @file
- * Install and update hooks for wri_io_content.
+ * Update hooks for wri_io_content
  */
 
 /**
- * Add the 'Show More Link' field to the International Office content type.
+ * Update the international_office display configs to remove blazy.
  */
 function wri_io_content_update_9301() {
 
+  \Drupal::service('distro_helper.updates')->installConfig('core.entity_view_display.node.international_office.card', 'wri_io_content', 'install', 'TRUE');
```

Comment:
Same thing here for the merge conflict resolution -- `Update the international_office display configs to remove blazy.` needs to be 9302.